src/main/java/com/automationpractice/pageobjects/ContactUsPage.java

```diff
@@ -0,0 +1,83 @@
+package com.automationpractice.pageobjects;
+
+import com.automationpractice.BasePage;
+import com.utils.ReadPropertyFile;
+import io.qameta.allure.Step;
+import org.apache.log4j.Logger;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+import java.io.File;
+
+public class ContactUsPage extends BasePage {
+
+    private WebDriver driver;
+    String baseUrl = ReadPropertyFile.getInstance().getContactUsPageUrl();
+    Logger log = Logger.getLogger("applicationLogger");
+
+    /*Constructor of current ContactUsPage class*/
+    public ContactUsPage(WebDriver driver) {
+        this.driver = driver;
+        PageFactory.initElements(driver, this);
+    }
+
+    @FindBy(xpath = "//h1[contains(@class, 'page-heading')]")
+    private WebElement contactUsPageHeading;
```

Comment:
For elements I recommend you to add prefixes about element meaning e.g. lblContactUsPageHeading
You can setup your own convention or use e.g. [Object Naming Conventions](https://docs.microsoft.com/en-us/previous-versions/visualstudio/visual-basic-6/aa263493(v=vs.60)?redirectedfrom=MSDN)